Update db-active-record.md

pixlmonstr opened the pull request:
Fixed syntax error in PHP example

pixlmonstr committed:
Update db-active-record.md

Fixed syntax error in PHP example

cebe commented on docs/guide/db-active-record.md:
```diff
@@ -667,7 +667,7 @@ public function actionUpdate($id)
     try {
         if ($model->load(Yii::$app->request->post()) && $model->save()) {
-            return $this->redirect(['view', <?= $urlParams ?>]);
+            return $this->redirect(['view', $urlParams]);
```
should be something like `['id' => $model->primaryKey]` instead of `$urlParams` because this variable is not defined.

pixlmonstr replied:
Yes, you are right. I just starred on the failed syntax highlighting.

pixlmonstr committed:
Update db-active-record.md

Added cebe's approach of displaying the newly created content after saving it to the db.

qiangxue commented:
Thanks!
